chapter4.adoc

```diff
@@ -59,10 +59,17 @@ The following MTT L3 entry accomodates a 56 bit PAW:
 ....
 {reg: [
   {bits:  44, name: 'MTTL2 PPN'},
-  {bits:  20, name: 'ZERO'},
+  {bits:  19, name: 'ZERO'},
+  {bits:   1, name: 'VALID'},
```

Comment:
Any reason to not use the same bit positions that MTTL2 uses for `TYPE`?

Reply:
can do